pygments/formatters/latex.py

```diff
@@ -287,19 +287,29 @@ def rgbcolor(col):
             cmndef = ''
             if ndef['bold']:
                 cmndef += r'\let\$$@bf=\textbf'
+            else:
+                cmndef += r'\let\$$@bf=\relax '
             if ndef['italic']:
                 cmndef += r'\let\$$@it=\textit'
+            else:
+                cmndef += r'\let\$$@it=\relax '
             if ndef['underline']:
                 cmndef += r'\let\$$@ul=\underline'
+            else:
+                cmndef += r'\let\$$@ul=\relax '
             if ndef['roman']:
                 cmndef += r'\let\$$@ff=\textrm'
-            if ndef['sans']:
+            elif ndef['sans']:
                 cmndef += r'\let\$$@ff=\textsf'
-            if ndef['mono']:
+            elif ndef['mono']:
                 cmndef += r'\let\$$@ff=\textsf'
```

Comment:
If this is mono, should this be `\texttt`?

Reply:
I didn't touch that piece of the code, simply changed if for elif because the font can't be both serif and sans-serif at the same time.



It would indeed make sense (I saw another issue on this), but I am not sure how this option is used. In my case the output is always monospaced. Is this a legacy option no longer in use?

Reply:
I have no idea to be honest, I haven't tried the LaTeX formatter myself :) It just looked wrong to me. Maybe @birkenfeld remembers how this came to be?

Reply:
I also wonder why the `'mono'` does not trigger `\texttt`.

Reply:
Must be a copy-paste error from `sans` above. But I don't see `mono` being supported anywhere else, so it would be best to simply remove it, along with `roman` and `sans`.